CRM/Dedupe/BAO/DedupeRuleGroup.php

```diff
@@ -414,11 +414,15 @@ public function fillTable(int $id, array $contactIDs, array $params): void {
     // if there are no rules in this rule group
     // add an empty query fulfilling the pattern
     if (!$tableQueries) {
-      // Yeah not too sure why but ....,
+      // Just for the hook.... (which is deprecated).
       $this->noRules = TRUE;
     }
+    CRM_Utils_Hook::dupeQuery($this, 'table', $tableQueries);
```

Comment:
Sone change by shifting to here is that the temporayTables class var won't be set right?

Reply:
@seamuslee001 yeah - I think that's OK because

1) the hook appears to be unused & is being deprecated
2) the documentation on using the hook has always specified that the way to use it is to set the `tableQueries` parameter
https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_dupeQuery/

Reply:
3) temporaryTables is protected